guides-for-using-cinchy/additional-guides/cinchy-secrets-manager.md

```diff
@@ -2,95 +2,176 @@
+
+## Use a secret in real-time syncs
 
-<figure><img src="../../.gitbook/assets/image (4).png" alt=""><figcaption><p>Image 3: Use your Variable in a REST Header</p></figcaption></figure>
+You can also use your Cinchy Secrets when configuring your Listener for real-time syncs.
 
-## Use a Secret in real-time syncs
+To use a secret in real-time syncs:
 
-You are also able to use your Cinchy Secrets when configuring your Listener for real-time syncs.
+1. When configuring your sync, navigate to the **Info Tab > Variables**.
+2. Under the **Variables** section, choose **Secret**.
+3. Input the name of your variable.
+4. Under the **Value** dropdown, choose the secret you intend to assign from the **Secrets** table.
+5. Go to the **Source** tab.
+6. Within the **Listener** section, input the secret variables as values for the relevant property in your **Topic** or **Connection Attribute** fields.
+
+   For example:
+
+   ```json
+   {
+     "InstanceAuthUrl": "@Url",
+     "ApiVersion": 41.0,
+     "GrantType": "@GrantType",
+     "ClientId": "@ClientId",
+     "UserName": "@Username",
+     "Password": "@Password"
+   }
+   ```
 
-To use a secret in the Listener:
+## Use a secret in the Listener Config table
 
-1. When configuring your sync, navigate to the **Info Tab > Variables.**
-2. Under the **Variables** section, select **Secret.**
-3. Under **Listener** section, use your secrets in the **Topic** or **Connection Attributes** field of your sync. For example:
+You can also add a secret that's attached to a variable to the **Topic** or **Connection Attributes** in the Listener Config table.
+
+1. Open the **Listener Config** table.
+1. Select the row that corresponds to your data sync.
+1. Select the **Topic** or **Connection Attribute** cell you want to change.
+1. Replace the value for a property with the variable assigned to a secret.
+
+For example, in the JSON code below, the Connection Attribute property `connectionString` is replaced with the `@connectionString` variable defined in the data sync.
 
 ```json
-// Lists variables for GrantType, ClientID, Username, Password 
 {
-"InstanceAuthUrl": "@Url",
-"ApiVersion": 41.0,
-       "GrantType": "@GrantType",
-       "ClientId": "@ClientId",
-       "UserName": "@Username",
-       "Password": "@Password"
-   }
+  "connectionString": "@connectionString",
+  "retryConfiguration": {
+    "retryMaxAttempts": "2",
+    "retryDelayStrategy": "Linear"
+  }
+}
 ```
+
+## Listener Config parameters
+
+The following table provides an overview of which parameters you can use as secrets or variables for each event connector type.
+
+| Event Connector Type      | Topic                | Connection Attributes | Value as Parameter/Secrets |
+| ------------------------- | -------------------- | --------------------- | -------------------------- |
+| Cinchy CDC                | tableGuid            |                       | Yes                        |
```

Comment:
@balmasi If I use a secret for my tableGuid? Will the UI be able to "translate"/resolve that so that the list of tables (Cinchy Event Broker) is presented correctly?

Reply:
@cinchy-m  No... Resolution happens at runtime. The behaviour of the front-end component I think is current undefined

Reply:
I think we're gonna say not supported for the tableGuid

Reply:
So, the UI will work as expected, yeh?